keyboards/electronlab/klor/glcdfont.c

```diff
@@ -0,0 +1,233 @@
+// This is the 'classic' fixed-space bitmap font for Adafruit_GFX since 1.0.
+// See gfxfont.h for newer custom bitmap font info.
```

Comment:
This would need a proper copyright/license header.

Reply:
Agreed. Change is done and committed.